Sources/TextMessageCell.swift

```diff
@@ -70,6 +70,11 @@ open class TextMessageCell: MessageCollectionViewCell<MessageLabel> {
             messageContentView.text = text
         case .attributedText(let text):
             messageContentView.attributedText = text
+        case .emoji(let text):
+            messageContainerView.backgroundColor = .clear
```

Comment:
Yeah, I was thinking how to handle the style. Maybe we shouldn't set the color to `.clear` here and expect the user to apply a style of `.none` to the `.emoji` case.

Reply:
I was originally thinking the same thing, by just setting the style to .none here. But the ```MessageCollectionViewCell``` sets the background color from the delegate.

Reply:
Tough one. Can we just do this in the delegate then? You'll have to do an `if case let` or switch on the `MessageType`. If it is emoji return a `.clear` background color. Until we find a better way to handle it.

Reply:
Moved to the delegate.